Parser/Indent.swift

```diff
@@ -19,9 +19,13 @@ enum IndentType: Character {
 }
 
 class Indent {
-    static var char: String = ""
+    static var char: String = "" {
+        didSet {
+            size = char.characters.count
+        }
+    }
     static var size: Int = 0
-    static var paraAlign = true
+    static var paraAlign = false
```

Comment:
I think that settings should be off by default, in general. If someone wants Swimat to do something for them they should turn it on. This follows the principle of least surprise.